pyside6_overlay/ui/main_window.py

```
@@ -0,0 +1,311 @@
+    QSplitter, QMenuBar, QStatusBar, QApplication
+)
+from PySide6.QtCore import Qt, QTimer, Signal
+from PySide6.QtGui import QAction
+
+from qfluentwidgets import (
+    NavigationInterface, NavigationItemPosition, FluentIcon,
+    setTheme, Theme, isDarkTheme, qconfig
+)
+
+from .workspace_manager import WorkspaceManager
+from ..components.theme_manager import ThemeManager
```

Pylint (3.3.7)

[error] Attempted relative import beyond top-level package

(E0402)